scripts/run-travis-tests.sh

```diff
@@ -18,10 +18,9 @@ run() {
 }
 
 run pip_chainer sudo pip3 install third_party/chainer
-run pip_onnx_chainer sudo pip3 install third_party/onnx-chainer
-# TODO(hamaji): Remove this once ONNX-chainer becomes compatible with
-# ONNX-1.5.0.
-run pip_onnx sudo pip3 install onnx==1.4.1
+run pip_onnx_chainer sudo pip3 install third_party/onnx-chainer onnxruntime==0.4.0 onnx==1.5.0
```

Comment:
Using extra requirements like `pip install third_party/onnx-chainer[test-cpu]` can install appropriate onnxruntime and onnx, ref: https://github.com/chainer/onnx-chainer/blob/master/setup.py#L17

Especially onnxruntime is used only by onnx-chainer, so had better not to set version number on onnxruntime. onnx is used by some modules, so I have no idea to set version number or not here.